MediaBrowser.Model/System/SystemInfo.cs

```diff
@@ -3,6 +3,21 @@
 
 namespace MediaBrowser.Model.System
 {
+    /// <summary>
+    /// Enum describing the location of the FFmpeg tool.
+    /// NotFound: No path to FFmpeg found
+    /// External: Path supplied via command line using switch --ffmpeg
+    /// Custom: User has supplied path via Transcoding UI page
+    /// System: FFmpeg tool found on system $PATH
+    /// </summary>
+    public enum FFmpegLocation
+    {
+        NotFound,
+        External,
```

Comment:
as stated in other file, I think `External` is not descriptive enough, maybe something like `SetByArgument` would fit better